build.gradle

```diff
@@ -158,6 +158,7 @@ sonarqube {
         property "sonar.jacoco.itReportPath", "${project.buildDir}/jacoco/integration.exec"
         property "sonar.exclusions", "src/main/java/uk/gov/hmcts/reform/sscs/domain/**," +
                 "src/main/java/uk/gov/hmcts/reform/sscs/TribunalsCaseApiApplication.java," +
+                "src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SscsCaseDataWrapperDeserializer.java," +
```

Comment:
why are you excluding it?

Reply:
Serialisation logic has slightly different treatment of nulls

Reply:
Removed